src/static_handler.rs

```diff
@@ -16,7 +16,8 @@ use requested_path::RequestedPath;
 ///
 /// If the path doesn't match any real object in the filesystem, the handler will return
 /// a Response with `status::NotFound`. If an IO error occurs whilst attempting to serve
-/// a file, `FileError(IoError)` will be returned.
+/// a file, an IronError will be returned.
```

Comment:
The actual underlying type of the error is important since downcasting - an "`IoError` as an `IronError`" is probably the right terminology for now.